releasenotes/notes/fix-asm-standalone-propagation-7bdb5a040ef82bac.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    ASM: Fixed an issue where distributed tracing headers were improperly propagated when tracing was disabled. Now, headers are only propagated for traces with the _dd.p.appsec tag, regardless of the propagation style (e.g., TraceContext). Previously, this logic applied only to Datadog headers, potentially allowing invalid trace info to be injected or extracted.
```

Comment:
Just to clarify - previously we were only propagating ASM distributed tracing headers on Datadog headers regardless if tracing was enabled/disabled? Is this because `DatadogMultiHeaders._inject()` doesn't account for tracing being enabled/disabled?